manifests/generated/operator-csv.yaml.in

```diff
@@ -772,6 +791,14 @@ spec:
           verbs:
           - get
           - list
+        - apiGroups:
+          - subresources.kubevirt.io
+          resources:
+          - virtualmachineinstances/guestosinfo
+          - virtualmachineinstances/filesystemlist
+          - virtualmachineinstances/userlist
+          verbs:
+          - get
```

Comment:
yes, this happens because of our rbac merging. The operator has its own set of  permissions and in addition gets all permissions which the subcomponents have. There can therefore be duplicates but it is harmless.